syntax = "proto3";

message Empty {
}

message Features {
  bytes no2_data = 1;
  bytes pm10_data = 2;
  bytes pm25_data = 3;
}

service Databroker {
  rpc get_next(Empty) returns(Features);
}